webapp/graphite/metrics_filter.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+from django.conf import settings
+from graphite.metric_filter.models import UserMetricFilter
+from graphite.metric_filter.models import GroupMetricFilter
+
+
+def nodeFilter(user, nodePath):
+  if settings.ALLOW_METRICFILTER_BY_USER:
+    if user.id is not None:
+      filters = UserMetricFilter.getFilters(user.id)
+      if filters is not None:
+        for filter in filters.all():
+          matching = match(nodePath, filter)
+          if matching is not None:
+            return matching
+    for filters in GroupMetricFilter.getFiltersByUser(user):
+      if filters is not None:
+        for filter in filters.all():
+          if filter is not None:
+            matching = match(nodePath, filter)
+            if matching is not None:
+              return matching
+    return False
+  else:
+    return True
```

Comment:
Since there is no way to fall through from a return ie the `return matching` or `return False`, you can change:

```python
    return False
  else:
    return True
```

To simply:

```python
  return False
return True
```